src/test/java/gov/cabinetofice/gapuserservice/service/OneLoginServiceTest.java

```diff
@@ -138,9 +128,10 @@ void shouldReturnValidAuthToken() throws IOException, JSONException {
 
         JSONObject result = oneLoginService.getTokenResponse("dummyJwt", "dummyCode");
 
-        Assertions.assertEquals( result, expected);
+        Assertions.assertEquals(result, expected);
     }
 
+    // TEST THIS ONE!
```

Comment:
why has this test specifically been called out? Does it require further work?

Reply:
Apologies - this comment was left to remind me to test after making a change but is not needed now. Will remove.